### ISSUE TITLE
PairsDetail page throws exception

### ISSUE THREAD

**Issue by modersohn** [loopexchange/Lexplorer#2]
I've already posted this at fudgeys repo, but he told me you guys would be hosting Lexplorer from now on, so here goes:

Visiting e.g. https://lexplorer.io/pairs/1-6 results in an unhandled exception, i.e. crash.

Unfortunately I cannot reproduce this locally, so I would need the logs to see what's going on.

Could whoever has access possibly post a relevant extract from the logs here?

**Comment by leppaludi** [comment]
```
Microsoft.JSInterop.JSException: call to eval() blocked by CSP
 renderChart/options<@https://lexplorer.io/_content/Blazor-ApexCharts/js/blazor-apex-charts.js:191:79
renderChart@https://lexplorer.io/_content/Blazor-ApexCharts/js/blazor-apex-charts.js:190:28
```

**Comment by modersohn** [comment]
I'm surprised Content Security Policy is active at all? We don't have anything in _Layout.cshtml. Is this enforced somehow by the new docker host? I don't see anything in the <head> section, where it should be located according to https://learn.microsoft.com/en-us/aspnet/core/blazor/security/content-security-policy?view=aspnetcore-6.0

**Comment by modersohn** [comment]
Okay, I've now figured out that the CSP is defined by the Response Header - so this seems to be caused by the new hosting environment:

<img width="589" alt="image" src="https://user-images.githubusercontent.com/44807458/232315588-dee9417b-290e-4037-8abb-66b6e3f4c003.png">

If I add the same CSP to _Layout.cshtml I can reproduce the crash locally.

I suspected the eval is caused by the custom JS I provided as Formatter for the Y-axis labels, e.g. here: https://github.com/loopexchange/Lexplorer/blob/74727440260bb0cd083ff5153001806299536008/Lexplorer/Pages/PairsDetail.razor#L258

And indeed if I remove all the formatters in PairsDetail.razor the crashes are gone!
